### ISSUE TITLE
"npm publish" is happening from local machine as part of "grunt release-*"

### ISSUE THREAD

**Issue by jkamke** [SciSpike/yaktor#22]
Perhaps npm publish should be happening during the travis build by default.

**Comment by matthewadams** [comment]
Closed by cebcd2b.  Note that you have to use grunt's `-f` or `--force` option due to a warning when running `npm run ci`:
```
Running "shell:ci" (shell) task
the input device is not a TTY
Warning: Command failed: /bin/sh -c ./run.sh npm run ci
the input device is not a TTY
```
We'll see if we can fix that soon.

**Comment by matthewadams** [comment]
83160ca fixes the TTY issue mentioned above.
